tests/testthat/test-derive_var_anrind.R

```diff
@@ -94,7 +94,7 @@ test_that("missing `AVAL` is handled properly", {
   input <- select(expected_output, USUBJID:ANRHI)
```

Comment:
While we're updating the test file, run the "Format test_file test file" add-in to get the formatting up-to-date with the other tests, wondering if we need to add an additional test using `use_a1hia1o = TRUE` ?

Reply:
At top of test script add:
`# derive_var_anrind ----` 

then use the add-in and the formatting should follow, see [here](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#addin-admiraldevformat_test_that_file) for details